src/main/java/com/adobe/aem/compgenerator/models/Options.java

```diff
@@ -174,8 +177,16 @@ public void setSharedProperties(List<Property> sharedProperties) {
         this.sharedProperties = sharedProperties;
     }
 
-    @Override
-    public boolean isValid() {
-        return true;
-    }
+	public List<Tab> getTabs() {
```

Comment:
Please use spaces instead of tabs for all code spacing, to be consistent with the existing codebase.

Reply:
Sure, I will update using spaces.